.babelrc

```diff
@@ -1,4 +1,3 @@
 {
-  "presets": ["es2015", "stage-1"],
-  "plugins": ["syntax-flow", "transform-flow-strip-types"]
+  "presets": ["es2015", "stage-1", "flow"]
```

Comment:
Is the "flow" preset required?

Reply:
Good catch, https://github.com/ripple/ripple-lib/pull/834 will remove it